### PR TITLE
PXB-2570 Release notes for XtraBackup 8.0.26-18.0

### PR DESCRIPTION
**Pull request by nastena1606**
new file: source/release-notes/8.0/8.0.26-18.0.rst
modified:   source/conf.py
	modified:   source/release-notes.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Percona XtraBackup enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot tolerate
+long periods of downtime. Offered free as an open source solution, it drives down backup
+costs while providing unique features for MySQL backups.
+
+Improvements
+================================================================================
+
+* :jirabug:`PXB-2477`: xbcloud should retry on error and utilize incremental backoff (Thanks to Baptiste Mille-Mathias for reporting this issue)
+* :jirabug:`PXB-2580`: xbcloud: retry chunk-upload upon receiving the ``SSL connect`` error
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Add thanks to Tim Vaillancourt for providing the patch

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* :jirabug:`PXB-2554`: Add the logic to skip tests if XtraBackup was compiled under ASAN
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: remove it

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXB-2101`: The Prepare step fails due to duplicate sdi (Thanks to Fungo Wang for reporting this issue)
+* :jirabug:`PXB-1504`: The FIND_GCRYPT macro is broken (Thanks to Maxim Bublis for reporting this issue)
+* :jirabug:`PXB-1961`: The Prepare step of a full backup fails for encrypted tables with a generic error
+* :jirabug:`PXB-2585`: XtraBackup fails to backup archive rockdb wal files
+* :jirabug:`PXB-2551`: Fix ``gr_basic`` test to stop on timeout in innodb80
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: remove

**Review by patrickbirch — COMMENTED** [review]
Date is still August 14

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
+.. _PXB-8.0.26-18.0:
+
+================================================================================
+*Percona XtraBackup* 8.0.26-18.0
+================================================================================
+
+:Date: August 14, 2021
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Date is wrong.

Comment (nastena1606) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Thanks. Fixed

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXB-2101`: The Prepare step fails due to duplicate sdi (Thanks to Fungo Wang for reporting this issue)
+* :jirabug:`PXB-1504`: The FIND_GCRYPT macro is broken (Thanks to Maxim Bublis for reporting this issue)
+* :jirabug:`PXB-1961`: The Prepare step of a full backup fails for encrypted tables with a generic error
+* :jirabug:`PXB-2585`: XtraBackup fails to backup archive rockdb wal files
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: RocksDB Write-Ahead Log (WAL) files

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+Percona XtraBackup enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot tolerate
+long periods of downtime. Offered free as an open source solution, it drives down backup
+costs while providing unique features for MySQL backups.
+
+Improvements
+================================================================================
+
+* :jirabug:`PXB-2477`: xbcloud should retry on error and utilize incremental backoff (Thanks to Baptiste Mille-Mathias for reporting this issue)
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: The xbcloud binary should retry on an error and utilize an incremental backoff

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 5 above)
Comment (altmannmarcelo) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: This was found out by a forum user. I'm waiting to get his name so we can credit him.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+* :jirabug:`PXB-2580`: xbcloud: retry chunk-upload upon receiving the ``SSL connect`` error
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: With the xbcloud binary, a chunk-upload on an SSL connect error to S3 was not retried.

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 3 above)
Comment (nastena1606) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Removed

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 2 above)
Comment (nastena1606) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Removed

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 7 above)
Comment (nastena1606) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Fixed

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 1 above)
Comment (nastena1606) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Added

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 6 above)
Comment (nastena1606) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Amended

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+for companies with large data sets and mission-critical applications that cannot tolerate
+long periods of downtime. Offered free as an open source solution, it drives down backup
+costs while providing unique features for MySQL backups.
+
+Improvements
+================================================================================
+
+* :jirabug:`PXB-2477`: The xbcloud should retry on error and utilize incremental backoff (Thanks to Baptiste Mille-Mathias for reporting this issue)
+* :jirabug:`PXB-2580`: With the xbcloud binary, a chunk-upload on an SSL connect error to S3 was not retried (Thanks to Tim Vaillancourt for providing the patch)
+* :jirabug:`PXB-2317`: Remove LOCKLESS binlog functionality
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Remove the LOCKLESS binary log functionality

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXB-2101`: The Prepare step fails due to duplicate sdi (Thanks to Fungo Wang for reporting this issue)
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Serialized Dictionary Information (SDI)

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+for companies with large data sets and mission-critical applications that cannot tolerate
+long periods of downtime. Offered free as an open source solution, it drives down backup
+costs while providing unique features for MySQL backups.
+
+Improvements
+================================================================================
+
+* :jirabug:`PXB-2477`: The xbcloud should retry on error and utilize incremental backoff (Thanks to Baptiste Mille-Mathias for reporting this issue)
+* :jirabug:`PXB-2580`: With the xbcloud binary, a chunk-upload on an SSL connect error to S3 was not retried (Thanks to Tim Vaillancourt for providing the patch)
+* :jirabug:`PXB-2317`: Remove the LOCKLESS binlog functionality
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: binary log instead of binlog.

**Review by patrickbirch — APPROVED** [review]
LGTM

**Review by george-lorch — APPROVED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 5 above)
Comment (altmannmarcelo) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: didn't get a reply on time :( . Let's proceed without the credits.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 10 above)
Comment (nastena1606) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Fixed

**Review by mchawla16 — APPROVED** [review]
Remove PXB-2526, if it's an internal issue and not impacting a customer

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst (hunk 8 above)
Comment (nastena1606) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.26-18.0.rst]: Fixed

**Pull request merged**